### PR TITLE
*: Catch up with refs/ removal (remove RefsRegexp, etc.)

### PR DESCRIPTION
**Pull request by wking**
Two minor fixups in the wake of #533.  Details in the commit messages (which I'm happy to split out into separate PRs if folks want).

### CODE REVIEW

File: image-layout.md [hunk 1]
```diff
@@ -58,7 +58,7 @@ A blob, referenced with digest `<alg>:<hex>` (per [descriptor](descriptor.md#dig
 The character set of the entry name for `<hex>` and `<alg>` MUST match the respective grammar elements described in [descriptor](descriptor.md#digests-and-verification).
 For example `sha256:5b` will map to the layout `blobs/sha256/5b`.
 
-The blobs directory MAY contain blobs which are not referenced by any of the refs.
+The blobs directory MAY contain blobs which are not referenced by any of the [refs](indexjson-file).
```
Comment (mikebrow) [inline image-layout.md]: "indexjson-file?"

Comment (wking) [inline image-layout.md]: Thanks.  Fixed with 9b434a7 → de2f01d.

**Comment by stevvooe** [comment]
LGTM

**Comment by vbatts** [comment]
LGTM

**Pull request merged**